docs/apache-airflow/start/docker.rst

```diff
@@ -49,8 +49,8 @@ Older versions of ``docker-compose`` do not support all the features required by
 
 .. warning::
     Default amount of memory available for Docker on MacOS is often not enough to get Airflow up and running.
-    If you have not enough memory available it might lead to airflow webserver continuously restarting.
-    You should have at least 4GB memory allocated for the Docker Engine (ideally 8GB). You can check
+    If enough memory is not allocated, it might lead to airflow webserver continuously restarting.
+    You should at least allocate 4GB memory for the Docker Engine (ideally 8GB). You can check
```

Comment:
FWIW I’ve been running Breeze with 3.9GB RAM (I allocate 4GB to WSL2 and the host system uses a little bit). Occassionally the webserver fails to allocate memory when I run `start-airflow` but generally works fine (the webserver usually works on second try).